include/boost/date_time/date_parsing.hpp

```diff
@@ -15,7 +15,6 @@
 #include <iterator>
 #include <algorithm>
 #include <boost/tokenizer.hpp>
-#include <boost/lexical_cast.hpp>
```

Comment:
So here we should ifdef the include so that if we're supporting pre-11 c++ we will just keep using lexical cast probably the cleanest solution is to move it to the compiler_config.hpp to do the macro check.  I looked and can't specific feature macro for these functions so I think I'd go with

BOOST_NO_CXX11

boost.config is already included so you should have direct access to this.

Reply:
Excellent!
...clarifying question...
Is the goal to totally eliminate the dependency on boost/lexical_cast?  If so then we could use ```cstdlib``` for ```< c++11``` and ```string``` for ```>= c++11```.
Or is the goal simply to modernize the code?  If so we can use the macro approach you mentioned for ```< c++11``` and leave the dependency on ```boost::lexical_cast```.

Reply:
I'm fine sticking with lexical_cast for 03 compiles and just modernizing for 11 and beyond.  It seems likely I'll eventually drop 03 support.